tests/utils/inputparams/testparams.go

```diff
@@ -23,8 +23,17 @@ import (
 	"os"
 	"strconv"
 	"time"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/govc"
```

Comment:
@shuklanirdesh82 Is this a regression issue? I'd suggest to send a separate quick fix for this blocking CI issue.